webpack/components/Table/TableWrapper.js

```diff
@@ -184,7 +185,7 @@ const TableWrapper = ({
             />
           </FlexItem>
         }
-        {!disableSearch && !hideToolbar &&
+        {!disableSearch && !hideToolbar && hasViewPermissions &&
```

Comment:
In the past we've used prefixes like `display`, `show`, `disable`, etc. to hide items in TableWrapper when the user doesn't have permissions, whereas the variables that actually store/describe the permission values are left up in the components. I'd prefer to see that pattern continued here, for consistency..

Reply:
Updated to rename and use hideSearch. (was disableSearch)